modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -50,6 +50,8 @@ interface BaseEditorOptions {
   add_form_submit_trigger?: boolean;
   add_unload_trigger?: boolean;
   allow_conditional_comments?: boolean;
+  allow_extended_mathml_attributes?: string[];
+  allow_extended_mathml_elements?: string[];
```

Comment:
Sorry if the conversation was confusing. It's just "extended" not "allow extended". The idea is to match `extended_valid_elements`.

```suggestion
  extended_mathml_attributes?: string[];
  extended_mathml_elements?: string[];
```

Reply:
Usually I want options that opt out of security be prefixed with allow to make the user think a bit of what am I allowing here exactly and why are things denied. However I guess these option names are fine since our extended_valid_elements leta you also opt out of security we just need to make sure that we conver this in docs.